server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/ProtocolServerService.java

```diff
@@ -266,7 +266,7 @@ public synchronized ProtocolServer getValue() throws IllegalStateException {
       return extensionManager;
    }
 
-    public Transport getTransport() {
+   public Transport getTransport() {
```

Comment:
this can be removed from here (the field too)
As soon as you do `:stop-connector` and `:start-connector`, this class keeps pointing to the old instance. 
The metrics bytes read/written are never updated after the restart.

Reply:
Done !